### PR TITLE
[ADD] event badge module

### PR DESCRIPTION
**Pull request by GillesTephaneMeyomesse**
This module allows to add an oca member logo on the badge

### CODE REVIEW

File: oca_event_badge/README.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
+===================
+OCA Custom Settings
+===================
+
+This module allow to add specific tags for oca partners.
+These tags are printed on the badge attendee.
```
Comment (sbidoul) [inline oca_event_badge/README.rst]: Please use the new readme fragments: https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

File: oca_event_badge/data/res_partner_category.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        <field name="name">OCA Delegate</field>
+        <field name="color" eval="9"/>
+    </record>
+
+    <record id="res_partner_oca_member_category" model="res.partner.category">
+        <field name="name">OCA Member</field>
+        <field name="color" eval="10"/>
+    </record>
+
+</odoo>
```
Comment (sbidoul) [inline oca_event_badge/data/res_partner_category.xml]: remove this: the tags are manually added in the OCA database

File: oca_event_badge/i18n/en_US.po [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+#. module: oca_event_badge
+#: model:res.partner.category,name:oca_event_badge.res_partner_oca_delegate_category
+msgid "OCA Delegate"
+msgstr "OCA Delegate"
+
+#. module: oca_event_badge
+#: model:res.partner.category,name:oca_event_badge.res_partner_oca_member_category
+msgid "OCA Member"
+msgstr "OCA Member"
```
Comment (sbidoul) [inline oca_event_badge/i18n/en_US.po]: no need for this translation file

File: oca_event_badge/views/event_event_templates.xml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+                    <span>
+                        <t t-esc="o.partner_id.parent_id.name"/>
+                    </span>
+                </div>
+            </div>
+
+            <div class="row">
+                <div t-if="bool(len(o.partner_id.category_id))" class="col-xs-12 text-center mt16" t-ignore="true">
+                    <span class="text-center" t-foreach="o.partner_id.category_id" t-as="val">
+                        <t t-if="val.name == 'OCA Board'">
```
Comment (sbidoul) [inline oca_event_badge/views/event_event_templates.xml]: val.name == 'board'

(resp 'member', 'delegate')

**Comment by GillesTephaneMeyomesse** [comment]
@sbidoul could you review these fixes pls ?

File: oca_event_badge/DESCRIPTION.rst [hunk 5]
```diff
@@ -0,0 +1,2 @@
+This module allow to add specific tags for oca partners.
```
Comment (elicoidal) [inline oca_event_badge/DESCRIPTION.rst]: This module allows to add specific tags for OCA partners.

File: oca_event_badge/USAGE.rst [hunk 6]
```diff
@@ -0,0 +1,4 @@
+To use this module, you need to:
+
+#. create tags in partner form ('board', 'member', 'delegate')
```
Comment (elicoidal) [inline oca_event_badge/USAGE.rst]: usually a list with one entry means that you do not need the list :wink: 

->  `To use this module, you need to create the tags in partner form ('board', 'member', 'delegate')`

File: oca_event_badge/__manifest__.py [hunk 7]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'OCA Event Badge',
+    'summary': """
+        Custom event badge add partner tag""",
```
Comment (elicoidal) [inline oca_event_badge/__manifest__.py]: "Creates Custom Event Badges based on the Partner Tags"

File: oca_event_badge/__manifest__.py [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+    'version': '11.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'website': 'https://acsone.eu',
```
Comment (elicoidal) [inline oca_event_badge/__manifest__.py]: repo URL

File: oca_event_badge/__manifest__.py [hunk 9]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline oca_event_badge/__manifest__.py]: no need coding

File: oca_event_badge/static/src/css/event_event_template.css [hunk 10]
```diff
@@ -0,0 +1,5 @@
+.badge_logo {
```
Comment (elicoidal) [inline oca_event_badge/static/src/css/event_event_template.css:4]: add copyright

**Review by elicoidal — APPROVED** [review]
some details. Needs test (no runbot): could you post the screenshot of the badge in a comment

**Comment by GillesTephaneMeyomesse** [comment]
@elicoidal Thanks for review

**Pull request merged**